recipes/fontconfig/fcf.patch

```diff
@@ -0,0 +1,15 @@
+diff --git src/fccfg.c src/fccfg.c
+index 6377fd7..8175f85 100644
+--- src/fccfg.c
++++ src/fccfg.c
+@@ -1830,9 +1830,7 @@ DllMain (HINSTANCE hinstDLL,
+ 
+ #endif /* !_WIN32 */
+ 
+-#ifndef FONTCONFIG_FILE
+-#define FONTCONFIG_FILE	"fonts.conf"
+-#endif
++#define FONTCONFIG_FILE	"@PREFIX@/etc/fonts/fonts.conf"
```

Comment:
This is needed to avoid an error shown in the terminal when other programs load fontconfig dynamic library.